src/manifold/include/manifold.h

```diff
@@ -251,6 +251,13 @@ class Manifold {
   static Manifold Hull(const std::vector<glm::vec3>& pts);
   ///@}
 
+  /** @name Minkowski Functions
+   */
+  ///@{
+  static Manifold Minkowski(const Manifold& a, const Manifold& b,
+                            bool inset = false, bool useThreading = false);
```

Comment:
I think if we get this all working, I'd prefer a member function, probably up in the Boolean section. And I presume `useThreading` is temporary until we figure out which way is best?

Reply:
Alrighty; I can switch it for a member function 👍 

And yeah; I see pretty similar performance from both the threaded and non-threaded versions, which makes me think most of the cost is in the Boolean at the end.    Once the triangulator/boolean robustness behavior is figured out (and maybe merging in the switch to `double`s?), I can settle for one version or the other...

Reply:
Actually, the threaded version now appears to be failing tests... this makes the decision easy, since it was only ever 1-2% speed difference (and sometimes slower; that close to the noise floor).

I've removed all of the references to `useThreading`.